modules/assim.batch/R/pda.emulator.R

```diff
@@ -404,10 +407,14 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
   # prepare for parallelization
   dcores <- parallel::detectCores() - 1
   ncores <- min(max(dcores, 1), settings$assim.batch$chain)
-  cl <- parallel::makeCluster(ncores, type="FORK")
+  
+  logger.setOutputFile(paste0(settings$outdir, "/pda.log"))
+  
   current.step <- "pre-MCMC"
   save(list = ls(all.names = TRUE),envir=environment(),file=pda.restart.file)
   
+  cl <- parallel::makeCluster(ncores, type="FORK", outfile = "pda.log")
```

Comment:
You're sure `pda.log` will be in the right directory? Might be better to give an absolute path.